users-profile/README.md

```diff
@@ -77,6 +78,19 @@ go run .
 > Keploy Port: http://localhost:6789/testlist
 
 To generate Test Cases, you need to make some API calls. It could be using Thunder Client, Postman Desktop Agent, or your preferred API testing tool.
+here you can use this object template below for testing:
+```shell
+you can copy paste this object template:
+{
```

Comment:
@Sekhar-Kumar-Dash this needs to be changed to a valid curl request.

Reply:
@nehagup  I made the changes please review it again